articles/data-factory/connector-troubleshoot-guide.md

```diff
@@ -663,6 +663,27 @@ busy to handle requests, it returns an HTTP error 503.
 
 - **Recommendation**:  Check the sink dataset and fix the path without wildcard value.
 
+### Error code:  RemotePrivateLinkServiceSubscriptionNotRegistered
+- **Message**: `Failed to create PrivateEndpoint for client xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx, cluster : Id=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx, Name=. ReasonPhrase:Bad Request. Body:Microsoft.Azure.Management.Network.Fluent.Models.Error. Status code:BadRequest.`
+
+- **Cause**: Managed Private Endpoint provisioning status shows "Failed". This issue happens when the resource provider Microsoft.Network is not registered in the subscription.  Register  your subscription with Microsoft.Network resource provider.
+
+- **Recommendation**:  Refer link: https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types#register-resource-provider
+
+- 
+### Error Message: Unable to list blobs. Please try again.
```

Comment:
Per my understanding, this error is specific to event trigger creation, we should have a separate article for trigger troubleshooting, as well as add clarification on the symptom - upon which operation customer see this error message. The cause didn't mention it's trigger related.